### ISSUE TITLE
Clicking on rental property in edit mode

### ISSUE THREAD

**Issue by knownasilya** [ef4/stupendous-rentals#4]
Should clicking this do anything? Seems to be clickable but does nothing. This is on the main page.

**Comment by ef4** [comment]
Yes, we should probably navigate you to the content's own URL. This would let an editor navigate to content that doesn't actually expose a user-clickable link.

(For example, a "banner ad" content type may actually take users out of the app when they click it, but in editing mode you would instead want to navigate to the ad's "own page" where you can edit it and get more information about it.)

**Comment by ef4** [comment]
This fix belongs in the cardstack tools package. 

The functionality will also depend on the routing package but should degrade gracefully and do nothing is routing is not present. There are [already examples](https://github.com/cardstack/cardstack/blob/4d93dd717d3eb6616e37c2ae2362dd1acd9113a9/packages/tools/addon/components/cardstack-toolbox.js#L8) of places that behave this way.

**Comment by knownasilya** [comment]
I don't see that routing service being used in that example.

On the other hand, if there is no public page for the content type, then is there a default edit page generated by cardstack that can be routed to?

**Comment by ef4** [comment]
That component's template is passing the service down into some other components where it's actually used.

Yes, every piece of content necessarily has a canonical URL. It may have authorization restrictions, but it should always exist.
